src/web/test/integration/extension.test.ts

```diff
@@ -0,0 +1,20 @@
+import * as assert from 'assert';
+
+// You can import and use all API from the 'vscode' module
+// as well as import your extension to test it
+import * as vscode from 'vscode';
+// import * as myExtension from '../../extension';
+
+suite('Web Extension Test Suite', () => {
+  vscode.window.showInformationMessage('Start all tests.');
+
+  test('Sample test', () => {
+    assert.strictEqual(-1, [1, 2, 3].indexOf(5));
+    assert.strictEqual(-1, [1, 2, 3].indexOf(0));
+  });
+
+  test('Foo', () => {
+    assert.strictEqual(-1, [1, 2, 3].indexOf(5));
+    assert.strictEqual(-1, [1, 2, 3].indexOf(0));
+  });
```

Comment:
Is there any benefit in keeping these "sample" tests around?

Reply:
The reason it's a trivial test is that we don't actually have any functionality other than syntax highlighting to 'test'.

The main benefit to do this at all is that we have *something* here to run a unit test, so we can test that we can test the extension under the web host. If we add more functionality at a later date, the functionality is already present and we don't have to consult vscode docs to figure out how to add it, since it was already such a pain to adapt it to our current test harness.

I do think I should probably add this as a comment on the test, since I explained this to Daniel with our 1:1 pairing on this ticket.

Reply:
I understand there's nothing to test, but I still don't understand under what scenario would the test fail, i.e. what can we catch here? Is it that `activate()` will not throw exceptions?

Reply:
We are testing that we can activate under the web host. We aren't testing any functionality at this poit.

Reply:
@radeksimko are you OK with leaving this as is since getting the testing framework required so much work, or do you want to remove the entire unit test setup and ticket adding it back at a later date?

Reply:
I'm fine with it as-is, as long as we add a comment to say that we're merely testing successful activation, to prevent confusion of future readers. 😉

Reply:
Documented in a code comment inside the test